webapp/src/main/resources/public/js/utils/LinkHelper.js

```diff
@@ -14,7 +14,7 @@ class LinkHelper {
         let url = TemplateHelper.renderTemplate(urlTemplate, paramsWithEncoded);
 
         if (url) {
-            linkOrLabel = <a href={url}>{label}</a>;
+            linkOrLabel = <a href={url} target="_blank">{label}</a>;
```

Comment:
knit: not sure about this change, i'd just keep the default behavior and expect people to use shift+click if they want the new tab. In any case, this could be a different PR.

Reply:
Removed the target here and left it as it was previously